### PR TITLE
New heading for MCO release note

### PR DESCRIPTION
**Pull request by mburke5678**
I noticed that I wrote the [release note for this feature previously](https://github.com/openshift/openshift-docs/pull/45855/files#diff-c77c0cbc134fbd6d7bfb05509f65f4077fbbe88a6ce62ddafb44edc35e479f26R132) but forgot. I just merged a [different PR for this relase note](https://github.com/openshift/openshift-docs/pull/47929/files#diff-c77c0cbc134fbd6d7bfb05509f65f4077fbbe88a6ce62ddafb44edc35e479f26R440). 

I like the previous heading better.

### CODE REVIEW

File: release_notes/ocp-4-11-release-notes.adoc [hunk 1]
```diff
@@ -437,7 +437,7 @@ The Machine Config Operator (MCO) now updates the affected nodes alphabetically
 For more information, see xref:../post_installation_configuration/machine-configuration-tasks.html#machine-config-overview-post-install-machine-configuration-tasks[Machine config overview].
 
 [id="ocp-4-11-mco-signer-ca-alert"]
```
Comment (maxwelldb) [inline release_notes/ocp-4-11-release-notes.adoc:439]: Do you want to keep this ID?

File: release_notes/ocp-4-11-release-notes.adoc [hunk 2]
```diff
@@ -437,7 +437,7 @@ The Machine Config Operator (MCO) now updates the affected nodes alphabetically
...
-==== New alert if MCO cannot update the signer CA on a paused node
+==== Enhanced notification for paused Machine Config Pools upon certificate renewal
 
 You will now receive alerts in the Alerting UI of the {product-title} web console if the MCO attempts to renew an expired `kube-apiserver-to-kubelet-signer` CA certificate on a Machine Config Pool (MCP) that is paused. If the MCPs are paused, the MCO cannot push the newly rotated certificates to those nodes, which can result in failures.
```
Comment (maxwelldb) [inline release_notes/ocp-4-11-release-notes.adoc:442]: * If "Alerting" is a UI element or label, it should be bold, IIRC. 
* Should Machine Config Pool be "machine config pool"? That's how we have it elsewhere in the docs per https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#general-references

**Review by maxwelldb — COMMENTED** [review]
Looking pretty good. Not in scope comments, only. :) 

Generated preview locally.

File: release_notes/ocp-4-11-release-notes.adoc (hunk 2 above)
Comment (mburke5678) [reply release_notes/ocp-4-11-release-notes.adoc:442]: @maxwelldb Thanks for the review. 

Alerting UI is not bolded in the docs.
Maching config pool is lower-case, yes.

**Review by maxwelldb — APPROVED** [review]

**Pull request merged**